apm-agent-attach-cli/src/main/java/co/elastic/apm/attach/AgentAttacher.java

```diff
@@ -503,15 +503,15 @@ static void printHelp(PrintStream out) {
             out.println("        A list of PIDs to include.");
             out.println();
             out.println("    --include-main/--exclude-main <pattern>...");
-            out.println("        A list of regular expressions of fully qualified main class names or paths to JARs of applications the java agent should be attached to.");
+            out.println("        A regular expression of fully qualified main class names or paths to JARs of applications the java agent should be attached to.");
             out.println("        Performs a partial match so that `foo` matches `/bin/foo.jar`.");
             out.println();
-            out.println("    --include-vmarg/--exclude-vmarg <pattern>...");
-            out.println("        A list of regular expressions matched against the arguments passed to the JVM, such as system properties.");
+            out.println("    --include-vmargs/--exclude-vmargs <pattern>...");
```

Comment:
The UI and the server are using `vmarg` now. Maybe we should just change that in the attacher to match the docs, too. Or document `vmarg` and accept both `vmarg` and `vmargs`.

Reply:
I was not aware of that, I'll make sure our documentation is consistent with only `vmarg` and change the attacher to allow both `vmarg` and `vmargs` (and treat them the same) so to not make it a breaking change

Reply:
The apm-server config has been modified to only accept `vmargs` right now, since that is correct for the version bundled with the apm-server for 8.0. Can you open an issue in the apm-server when this change happens, so we can update the code there?